includes/core/server/request/class-get-account.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Class file for WCPay\Core\Server\Request\Get_Account.
+ *
+ * @package WooCommerce Payments
+ */
+
+namespace WCPay\Core\Server\Request;
+
+use WCPay\Core\Server\Request;
+use WC_Payments_API_Client;
+
+/**
+ * Request class for getting intents.
+ */
+class Get_Account extends Request {
+	use Use_Test_Mode_Only_When_Dev_Mode;
```

Comment:
Given that, we're having 4 requests like this under the current class `WC_Payments_API_Client`, I am approaching with this `trait`.